project_3.html

```diff
@@ -0,0 +1,407 @@
+			</p>
+
+			<h4 class="mb-3 fw-light" id="pj-heading-compiler">Compilers, Assemblers, and Linkers. Oh My!</h4>			
+			<p class="mb-3">
+				You write code in a programming language like C++, but that is not what the computer executes as instructions. The computer sees only 1s and 0s, or binary <strong>machine code</strong>. So how do we bridge the communication gap between you and the computer? We use a program called a <strong>compiler</strong> to translate your C++ code into machine language.
+                <br><br>
+                We tend to use “compiler” as an umbrella term to describe what is actually several programs working together to create machine code. In reality, the <strong>compiler</strong> turns a high-level language like C++ into assembly code. 
+                <br><br>
+                <strong>Assembly language</strong> is a low-level programming language based on a computer’s processor. Each assembly statement corresponds directly with a machine instruction or set of instructions. There was a time when most programmers wrote in assembly language; this is no longer commonplace.
+                <br><br>
+                The <strong>assembler</strong> converts the assembly language into machine code. We’re not done, though! One last program called a <strong>linker</strong> takes the machine code files and “links” them together into an <strong>executable file</strong>. You might recognize executables as files with the “.exe” extension on Windows. The executable is ultimately what the computer uses to execute your program.
+                <br><br>
+            </p>
+			<p class="mb-5" style="text-align:center;">
+				<img class="img-fluid my-3 me-5" width="300"
+				src="./assets/images/project_3/[2023-24] OPS Project 3_ C++ Programming (1).png">
+			</p>
+
+
+			<h2 class="mb-3 fw-bold" id="pj-heading-ide-setup">IDE Setup</h2>
+			<p class="mb-3">
+				Before we get started, you will need to choose an IDE. It is recommended that you use Replit, a free online IDE, for this project.
+			</p>
+			<h4 class="mb-3 fw-light" id="pj-heading-replit">Replit</h4>
+			<p class="mb-3">
+				Visit this <a href="https://repl.it/">link</a> and create an account with Replit. You will notice that there are paid pricing options. However, they are not required to complete the project.
+			</p>
+			<p class="mb-5" style="text-align:center;">
+				<img class="img-fluid my-3 me-5"
+				src="./assets/images/project_3/replit_image.png">
+			</p>
+
+
+			<h2 class="mb-3 fw-bold" id="pj-heading-requirements">Requirements</h2>
+			<p class="mb-3">
+				You will write five functions into one .cpp file. The requirements for each function are listed below. They must pass the autograder when submitted.
+			</p>
+			<h4 class="mb-3 fw-light" id="pj-heading-echo">Echo</h4>
+			<p>
+				<ul class="mb-5">
```

Comment:
Please change the following lists to be in the Boostrap format (similar to the Requirements section list) If there's a second bullet pointed list embedded into the first, you can leave it plain (no Boostrap class).

Reply:
Hi Gavin, just to clarify, I meant to say the lists should be formatted similarly to the requirements sections of the other projects. Each list item is "boxed" in.

![image](https://github.com/uci-ieee/ops-webpages-2023-2024/assets/45702229/f04fe253-cfbd-4146-bdfa-6c5adf8f1b26)